Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_033.MongoDbCredentialsValidator.cs

```diff
@@ -19,45 +23,65 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                 return ValidationResult.CreateNoMatch();
             }
 
+            groups.TryGetValue("schema", out FlexMatch schema);
+            groups.TryGetValue("options", out FlexMatch options);
+
             var validationResult = new ValidationResult
             {
                 Fingerprint = new Fingerprint()
                 {
                     Id = id.Value,
                     Host = host.Value,
                     Secret = secret.Value,
+                    Scheme = schema?.Value,
+                    QueryString = options?.Value,
```

Comment:
> QueryString = options?.Value, [](http://example.com/codeflow?start=19&length=30)

Looking at fingerprint fields, is queryString the best fit? I ask because queryString was created to handle a special case in a different rule, so questioning its use here.

Reply:
then I should not use `QueryString`, how about `Part`?
this is the name for the options parts of the connection string 
e.g. "mongodb://user:password@example.com:10255/?ssl=true&retrywrites=false&replicaSet=globaldb&maxIdleTimeMS=120000", these key value pairs after port number are the options for the connection.

Reply:
No, I was just asking to ask, not asking for a change. After looking more, queryString is correct here. I'll try to be more explicit when I'm asking for a change vs just asking to be curious.